console/console-init/ui/src/modules/connection/components/ConnectionsToolbar/ConnectionsToolbar.tsx

```diff
@@ -57,6 +67,25 @@ const ConnectionsToolbar: React.FunctionComponent<IConnectionsToolbarProps &
     { key: "sender", value: "Senders", index: 5 },
     { key: "receiver", value: "Receivers", index: 6 }
   ];
+  const dropdownItems = [
+    <DropdownItem
+      id="cl-filter-dropdown-item-closeAll"
+      key="close-all"
+      value="closeAll"
+      component="button"
+      isDisabled={selectedConnections.length <= 0}
+    >
+      Close Selected
+    </DropdownItem>
+  ];
+
+  const onKebabSelect = async (event: any) => {
+    const value = event.target.value;
```

Comment:
As discussed, please rename 'value'

Reply:
It seems to be okay in this case because we are getting the value from the event and make more sense to me to be named as value.

Reply:
done